agent/agent.go

```diff
@@ -64,6 +76,17 @@ func Start() error {
 	return err
 }
 
+func configdir() (string, error) {
+	gopsdir := fmt.Sprintf("%s/.gops", os.Getenv("HOME"))
```

Comment:
Let's put things under .config/gops on Unix-like systems and under APPDATA on windows.

```
func configdir() (string, error) {
	if runtime.GOOS == "windows" {
		return filepath.Join(os.Getenv("APPDATA"), "gops"), nil
	}
	homeDir := guessUnixHomeDir()
	if homeDir == "" {
		return "", errors.New("unable to get current user home directory: os/user lookup failed; $HOME is empty")
	}
	return filepath.Join(homeDir, ".config", "gops"), nil
}

func guessUnixHomeDir() string {
	usr, err := user.Current()
	if err == nil {
		return usr.HomeDir
	}
	return os.Getenv("HOME")
}
```